### PR TITLE
6247 validation summary fix

### PR DESCRIPTION
**Pull request by RuthBerry96**

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Identity.Api/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -20,8 +20,15 @@
...
-
-    <title>@ViewData["PageTitle"]</title>
+    
+    @if (@ViewData.ModelState.IsValid)
+    {
+        <title>@ViewData["PageTitle"]</title>
+    }
+    else
+    {
+        <title>Error: @ViewData["PageTitle"]</title>
```
Comment (kat1906) [inline src/NHSD.BuyingCatalogue.Identity.Api/Views/Shared/_Layout.cshtml:30]: I think the 'Error' has to be hidden as it is for screen readers

Comment (kat1906) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Shared/_Layout.cshtml:30]: update: this is for the browser tab so is fine

File: src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/Login.cshtml [hunk 2]
```diff
@@ -1,48 +1,40 @@
...
+                            </div>
                         </div>
-                    </form>
-                </div>
-            </div>
+                    </div>
         </div>
     </main>
-</div>
+</div>
```
Comment (kat1906) [inline src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/Login.cshtml:40]: new line

Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/Login.cshtml:40]: 👍

**Pull request merged**